.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -70,46 +70,49 @@ Identify/create the bundles that we will release for Kata and enclave-cc.
     and that the operator pulls the most recent version of the container.
 
     * Check that the version of the `nydus-snapshotter` used by Kata matches the one used by the operator
-        * Compare `nydus-snapshotter` version in Kata [versions.yaml](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml#L291) with the [Makefile](https://github.com/confidential-containers/operator/blob/main/install/pre-install-payload/Makefile#L4) for the operator pre-intall container.
-            * If they do not match, update the operator. This can be part of the PR described in the next step.
-    
-    * Make sure that the operator pulls the most recent version of the pre-install container
-        * Find the last commit in the [pre-install directory](https://github.com/confidential-containers/operator/tree/main/install/pre-install-payload)
-            * Make sure that the commit matches the preInstall / postUninstall image specified for [enclave-cc CRD](https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml) and [Kata CRD](https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml)
-                * If these do not match (for instance if you changed the snapshotter above), update the operator so that they do match. This can be part of the PR described in the next step.
-    
+        * Compare `nydus-snapshotter` version in Kata [versions.yaml](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml#L325) with the [Makefile](https://github.com/confidential-containers/operator/blob/main/install/pre-install-payload/Makefile#L4) for the operator pre-intall container.
+            * **If they do not match, stop and open a PR now. In the PR, update the operator's Makefile to match the version used in kata. After the PR is merged, continue.**
 
 - [ ] 4. :wrench: **Open a PR to the operator to update the release artifacts**
 
-    Update the operator to use the payloads identified in steps 1 and 2.
+    Update the operator to use the payloads identified in steps 1, 2, and 3.
 
-    There are a number of places where the payloads are referenced. Make sure to update all of the following to the tag matching the latest commit hash from steps 7 and 8:
+    Make sure that the operator pulls the most recent version of the pre-install container
+        * Find the last commit in the [pre-install directory](https://github.com/confidential-containers/operator/tree/main/install/pre-install-payload)
+            * Make sure that the commit matches the preInstall / postUninstall image specified for [enclave-cc CRD](https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml) and [Kata CRD](https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml)
```

Comment:
I think the correct word is `ccruntime CDR` rather than `Kata CDR`.

Reply:
Maybe mention that the pre-install image is `quay.io/confidential-containers/reqs-payload`